### PR TITLE
docs: install boundary amazon linux instruction

### PR DESCRIPTION
**Pull request by DhariniJeeva**
jira- https://hashicorp.atlassian.net/browse/SPE-239
add amazon linux installation steps

Screenshot 2023-07-12 at 2 24 02 PM

### CODE REVIEW

File: website/content/docs/install-boundary/install.mdx [hunk 1]
```diff
@@ -70,4 +70,26 @@ Select your distribution, and complete the steps to install Boundary.
     ```
 
 </Tab>
-</Tabs>
\ No newline at end of file
+
+<Tab heading="Amazon">
```
Comment (AdamBouhmad) [inline website/content/docs/install-boundary/install.mdx:74]:
```suggestion
<Tab heading="Amazon Linux">
```

**Review by AdamBouhmad — CHANGES_REQUESTED** [review]
One note on changing the heading from Amazon to Amazon Linux to be a bit more explicit, otherwise this looks great!

**Review by AdamBouhmad — APPROVED** [review]
LGTM!

**Review by Dan-Heath — APPROVED** [review]
Looks great! Thanks so much for taking this one on. 🎉

**Pull request merged**